lesson01/assignment/index.html

```diff
@@ -12,14 +12,68 @@
 <body>
   <main>
     <h1>Using ES6+ Methods in DOM Interactions</h1>
-    <form id="example-form">
-      <label>Enter your favorite number:
-        <input type="text" class="input">
-      </label>
-      <input type="submit" value="See Demo!">
-      <p>The total of my array elements summed with your favorite number is: <span class="output"></span></p>
-    </form>
+    <section>
+      <h2>Example Form</h2>
+      <form id="example-form">
+        <label>Enter your favorite number:
+          <input type="text" class="input">
+        </label>
+        <input type="submit" value="See Demo!">
+        <p>The total of my array elements summed with your favorite number is: <span class="output"></span></p>
+      </form>
+    </section>
+
+    <section>
+      <h2>[1] ES6 .includes() Method</h2>
+      <form id="form1">
+        <label>Enter your an item to add to your grocery list:
+          <input type="text" class="input">
+        </label>
+        <input type="submit" value="See Demo!">
+        <p>Your Grocery List: <span class="output"></span></p>
+
+        <p>Does your list include eggs?  <span id="eggs"></span></p>
+
+      </form>
+    </section>
+
+    <section>
+      <h2>[2] ES6 .map() Method</h2>
+      <p>List of Numbers - 2 , 3 , 4 , 5 , 6</p>
```

Comment:
Consider populating this from the javascript so it always has the same value both places.